app/assets/scripts/map.js

```diff
@@ -4,9 +4,68 @@
 // <script src="https://maps.googleapis.com/maps/api/js?key=YOUR_API_KEY&libraries=places">
 
 var map;
+var oms;
 var markers = [];
 var infowindow;
 
+// Click listener for a marker.
+function markerListener(marker, event) {
```

Comment:
Then can we replace the 
`moreLink.addEventListener('click', function() {` with 
`moreLink.addEventListener(event, function() {` instead? 
and then make the call `markerListener(markerToShow, 'click')` on line 251?
So that we don't hardcode the event and make use of the variable?